lib/node_modules/@stdlib/blas/base/drot/README.md

```diff
@@ -0,0 +1,187 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# drot
+
+> Applies a plane rotation.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var drot = require( '@stdlib/blas/base/drot' );
+```
+
+#### drot( N, x, strideX, y, strideY, c, s )
+
+Applies a plane rotation.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var x = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+var y = new Float64Array( [ 6.0, 7.0, 8.0, 9.0, 10.0 ] );
+
+drot( x.length, x, 1, y, 1, 1.0, 0.0 );
+// x => <Float64Array>[ 1.0, 2.0, 3.0, 4.0, 5.0 ]
+// y => <Float64Array>[ 6.0, 7.0, 8.0, 9.0, 10.0 ]
+```
+
+The function has the following parameters:
+
+-   **N**: number of indexed elements.
+-   **x**: first input [`Float64Array`][mdn-float64array].
+-   **strideX**: index increment for `x`.
+-   **y**: second input [`Float64Array`][mdn-float64array].
+-   **strideY**: index increment for `y`.
+-   **c**: double precision cosine.
+-   **s**: double precision sin.
+
+The `N` and stride parameters determine how values in the strided arrays are accessed at runtime. For example, to apply plane rotation for the strided arrays
+in reverse order.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var x = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+var y = new Float64Array( [ 7.0, 8.0, 9.0, 10.0, 11.0, 12.0 ] );
+
+drot( x.length, x, 2, y, 2, 0.707, 0.707 );
+// x => <Float64Array>[ 5.656, 2.0, 8.484, 4.0, 11.312, 6.0 ]
+// y => <Float64Array>[ 4.242, 8.0, 4.241999999999999, 10.0, 4.241999999999999, 12.0 ]
+```
+
+Note that indexing is relative to the first index. To introduce an offset, use [`typed array`][mdn-typed-array] views.
+
+<!-- eslint-disable stdlib/capitalized-comments -->
+
```

Comment:
I guess you are missing an example, refer https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/blas/base/dswap/README.md?plain=1#L73C1-L88C4